### ISSUE TITLE
Faster ripping possible?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bleskodev/rubyripper#375]
```
This is a wish list ticket.

Ripping a CD takes fairly long. But I do not know what process takes so long. 
If I understand 
it correctly a track is read from the CD several times and then those version 
are compared.

Now looking at the CPU usage using `top` I do not see a process taking all CPU 
resources 
so I am wondering why not. Could someone help me.

On the other hand the next track is only read from CD when comparing the songs 
has been 
finished before. Could not all song be read in the background while the earlier 
once are still 
compared? So do more tasks in parallel?
```

Original issue reported on code.google.com by `pm.deb...@googlemail.com` on 23 Nov 2009 at 11:05
- Merged into: #348
